docs/android/README.md

```diff
@@ -6,32 +6,149 @@ PSPDFKit comes with open source wrappers for Cordova on both [iOS](https://pspdf
+- An [Android Virtual Device](https://developer.android.com/studio/run/managing-avds.html) or a hardware device.
 
-## Installation
+For more information regarding the Ionic installation you can check out the [official Ionic installation guide](https://ionicframework.com/docs/v1/guide/installation.html).
+
+## Cordova Installation
 
 We assume that you have [an existing Cordova project](https://cordova.apache.org/#getstarted).
 
-### Installation in a Cordova app
+1. Open the Terminal app and change the location of the current working directory inside the newly created project:
 
-```shell
+```bash
+cd path/to/YourProject
+```
+
+2. Remove all the platforms from your project to properly propagate the changes in the `config.xml` file below throughout the project:
+
+```bash
+cordova platform remove android
+cordova platform remove ios
+```
+
+3. Open `config.xml` in a text editor to enable AndroidX and to change the deployment target to iOS 12 or later:
+
+```bash
+open config.xml
+```
+
+Your `config.xml` file should look like this:
+
+```diff
+...
+  <platform name="android">
++   <preference name="AndroidXEnabled" value="true" />
+    <allow-intent href="market:*" />
+  </platform>
+  <platform name="ios">
+    <allow-intent href="itms:*" />
+    <allow-intent href="itms-apps:*" />
++   <preference name="deployment-target" value="12.0" />
+    ...
+  </platform>
+...
+```
+
+4. Add the PSPDFKit plugin:
+
+```bash
 cordova plugin add https://github.com/PSPDFKit/PSPDFKit-Cordova.git
 ```
 
-### Installation in an Ionic app
+5. Add back all the platforms:
 
-```shell
+```bash
+cordova platform add android
+cordova platform add ios
+```
+
+## Ionic Installation
+
+We assume that you have [an existing Ionic project](https://ionicframework.com/docs/cli/start/).
+
+1. Open the Terminal app and change the location of the current working directory inside the newly created project:
+
+```bash
+cd path/to/YourProject
+```
+
+2. Remove all the platforms from your project to properly propagate the changes in the `config.xml` file below throughout the project:
+
+```bash
+ionic cordova platform remove android
+ionic cordova platform remove ios
+```
+
+3. Open `config.xml` in a text editor to enable AndroidX and to change the deployment target to iOS 12 or later:
+
+```bash
+open config.xml
+```
+
+Your `config.xml` file should look like this:
+
+```diff
+...
+  <platform name="android">
+-   <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application" xmlns:android="http://schemas.android.com/apk/res/android">
+-      <application android:networkSecurityConfig="@xml/network_security_config" />
+-   </edit-config>
++   <preference name="AndroidXEnabled" value="true" />
+    ...
+  </platform>
+  <platform name="ios">
+    <allow-intent href="itms:*" />
+    <allow-intent href="itms-apps:*" />
++   <allow-navigation href="*" />
++   <preference name="deployment-target" value="12.0" />
```

Comment:
Nit: Only need the android ones for android.  I'm not sure it's worth changing though, but might be misleading for android devs.

```suggestion
```